index.js

```diff
@@ -1,5 +1,5 @@
-/**
- * Copyright (c) 2017, FinancialForce.com, inc
+/*
+ * Copyright (c) 2017-2020, FinancialForce.com, inc
```

Comment:
For future reference the rules on copyright notices have changed and so there is no need to do this anymore. Just the very first date on its own should do.